frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/operator.view.ts

```diff
@@ -202,6 +204,10 @@ export const operator = {
     nav.sidenav.clickNavLink(['Operators', 'Installed Operators']);
     cy.byTestOperatorRow(operatorName).should('not.exist');
   },
+  filterByName: (name: string) => {
+    cy.byTestID('name-filter-input').focus();
```

Comment:
We shouldn't need this: https://docs.cypress.io/api/commands/type#When-element-is-not-in-focus

Is there a bug here?

Reply:
The `input` is wrapped in `div` and doesn't get focus otherwise.  🤷